### PR TITLE
New package advisory for urgent crio fix

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: http://file.rdu.redhat.com/~choag/081120/rn-3.11-crio-update/release_notes/ocp_3_11_release_notes.html#ocp-3-11-248-upgrading

### CODE REVIEW

**Comment by mffiedler** [comment]
We can't point customers at mirror.openshift.com.  It is not accessible to them.  @vikaslaad Can you provide the correct location?

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -3453,6 +3454,19 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
...
+link:https://access.redhat.com/errata/RHBA-2020:3416[RHBA-2020:3416], complete
+the following steps:
+
+.  Point your yum repos to the latest mirror, like https://mirror.openshift.com/enterprise/all/3.11/latest/RH7-RHAOS-3.11/x86_64/os/), and run `yum clean all` on all nodes.
+
+. Run the xref:../upgrading/automated_upgrades.adoc#upgrading-ocp[3.11 upgrade playbook].
+
+. Verify all nodes get the `cri-o-1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7`
+package.
+
```
Comment (mffiedler) [inline release_notes/ocp_3_11_release_notes.adoc:3469]: Add a step to reboot all nodes

**Comment by vikaslaad** [comment]
@codyhoag could you please change the repo location text to "latest 3.11 yum repository"

it should read "Point your yum repos to the latest yum repository and run `yum clean all` on all nodes"

**Comment by vikaslaad** [comment]
@codyhoag this workaround is needed only if customers are running into https://bugzilla.redhat.com/show_bug.cgi?id=1866132, you can mention that as well.

**Review by mffiedler — APPROVED** [review]
/lgtm

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -3453,6 +3454,27 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
...
+. Clean all nodes:
++
+[source,terminal]
+----
+# yum clean all
+----
+
+. Run the xref:../upgrading/automated_upgrades.adoc#upgrading-ocp[3.11 upgrade playbook].
+
+. Verify all nodes get the `cri-o-1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7`
```
Comment (mburke5678) [inline release_notes/ocp_3_11_release_notes.adoc:3473]: @codyhoag Not sure how to reword, but _get_ seems odd here.
_Verify that the  `cri-o-1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7` package was downloaded to each node_?

Do I need to verify the nodes got the second package?

**Comment by mburke5678** [comment]
@codyhoag A couple things. Otherwise LGTM

File: release_notes/ocp_3_11_release_notes.adoc (hunk 2 above)
Comment (mffiedler) [reply release_notes/ocp_3_11_release_notes.adoc:3473]: yum list cri-o
verify version is 1.11.16-0.11.dev.rhaos3.11.gitd6a416d.el7

Comment (codyhoag) [reply release_notes/ocp_3_11_release_notes.adoc]: Just updated, Thanks!

**Pull request merged**